var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -0,0 +1,197 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+
+
+class Cantera(Package):
+    """Cantera is a suite of object-oriented software tools for problems
+    involving chemical kinetics, thermodynamics, and/or transport processes."""
+
+    homepage = "http://www.cantera.org/docs/sphinx/html/index.html"
+    url      = "https://github.com/Cantera/cantera/archive/v2.2.1.tar.gz"
+
+    version('2.2.1', '9d1919bdef39ddec54485fc8a741a3aa')
+
+    variant('lapack',     default=True,  description='Build with external BLAS/LAPACK libraries')
+    variant('threadsafe', default=True,  description='Build threadsafe, requires Boost')
+    variant('sundials',   default=True,  description='Build with external Sundials')
+    variant('python',     default=False, description='Build the Cantera Python module')
+    variant('matlab',     default=False, description='Build the Cantera Matlab toolbox')
+
+    # Required dependencies
+    depends_on('scons')
+
+    # Recommended dependencies
+    depends_on('blas',      when='+lapack')
+    depends_on('lapack',    when='+lapack')
+    depends_on('boost',     when='+threadsafe')
+    depends_on('sundials',  when='+sundials')  # must be compiled with -fPIC
+
+    # Python module dependencies
+    extends('python', when='+python')
+    depends_on('py-numpy',  when='+python')
+    depends_on('py-scipy',  when='+python')
+    depends_on('py-cython', when='+python')
+    depends_on('py-3to2',   when='+python')
+    # TODO: these "when" specs don't actually work
+    # depends_on('py-unittest2',     when='+python^python@2.6')
+    # depends_on('py-unittest2py3k', when='+python^python@3.1')
```

Comment:
can you add another issue for this one?  Just `^python@2.6` should be sufficient (if `+python` implies `^python`).  That should work but I think we should address it in a separate PR.